test/test_rpc.py

```diff
@@ -90,7 +92,17 @@ def wrapper(self):
     sys.version_info < (3, 0),
     "Pytorch distributed rpc package " "does not support python2",
 )
+@unittest.skipIf(C.is_asan, "Skip ASAN since torch + multiprocessing + fbcode doesn't work with asan")
 class RpcTest(MultiProcessTestCase):
+    @classmethod
+    def setUpClass(cls):
+        super(RpcTest, cls). setUpClass()
+        try:
+            multiprocessing.set_start_method("spawn")
+        except Exception:
```

Comment:
Why do we ignore this exception and continue? We should throw this exception and fail the test.

Reply:
for some env, if there is no spawn, at least we should run fork?